android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -142,4 +236,449 @@ public void rejectEvent(String eventId) {
         Radar.rejectEvent(eventId);
     }
 
+    @ReactMethod
+    public void getContext(final Promise promise) {
+        if (promise == null) {
+            return;
+        }
+
+        Radar.getContext(new Radar.RadarContextCallback() {
+            @Override
+            public void onComplete(@NonNull Radar.RadarStatus status, @Nullable Location location, @Nullable RadarContext context) {
+                if (status == Radar.RadarStatus.SUCCESS) {
+                    try {
+                        WritableMap map = Arguments.createMap();
+                        map.putString("status", status.toString());
+                        if (location != null) {
+                            map.putMap("location", RNRadarUtils.mapForJson(Radar.jsonForLocation(location)));
+                        }
+                        if (context != null) {
+                            map.putMap("user", RNRadarUtils.mapForJson(context.toJson()));
```

Comment:
nit: the top-level key we use in the API responses is `context` - https://github.com/radarlabs/server/blob/master/app.ts#L647

might be confusing if we key by `user` if this is meant to be a stateless operation

Reply:
Good catch, this should be `context`. Fixing.